dkg-runtime-primitives/src/proposal.rs

```diff
@@ -30,52 +27,98 @@ pub use webb_proposals::{
 
 #[derive(Clone, RuntimeDebug, scale_info::TypeInfo)]
 pub struct RefreshProposal {
+	/// The merkle root of the voters (validators)
+	pub voter_merkle_root: [u8; 32],
+	/// The session length in milliseconds
+	pub session_length: u64,
```

Comment:
If the session length is in milliseconds, do we need 64-bits to store that data, or, can we opt for something lower?

```suggestion
	pub session_length: u32,
```

Reply:
I think that `u64` is good. With a `u32` we are limited to a maximum session length of around 49-50 days I believe. Calculated as $50 * 3600 * 24 * 1000 = 4320000000 > 4294967296$